src/components/posts/helperfunctions.js

```diff
@@ -0,0 +1,22 @@
+import { differenceInHours, differenceInDays } from 'date-fns'
+
+function dateTimetoRelativeTime(datetime){
```

Comment:
Is it not possible to use the `formatDistance` function from this lib? See: https://date-fns.org/

```javascript
formatDistance(subDays(new Date(), 3), new Date(), { addSuffix: true })
//=> "3 days ago"
```

Or is that not suitable for what we need?

Reply:
Wasn't aware of this library function, looks as though it may be ideal!